Source/WebCore/dom/NodeIterator.cpp

```diff
@@ -123,7 +125,7 @@ ExceptionOr<RefPtr<Node>> NodeIterator::previousNode()
     RefPtr<Node> result;
 
     m_candidateNode = m_referenceNode;
-    while (m_candidateNode.moveToPrevious(root())) {
+    while (m_candidateNode.moveToPrevious(protectedRoot())) {
```

Comment:
Given that this is a loop, do we want to cache `protectedRoot()` in a variable before the loop to avoid ref counting churn?

Reply:
That's a good point. Will address this in https://github.com/WebKit/WebKit/pull/19129